src/security-events/EventMetadata.ts

```diff
@@ -0,0 +1,97 @@
+import {clearUndefinedProperties} from "../Util";
+
+export interface EventMetadataJson {
+    tenantId: string;
+    timestampMillis: number;
+    iclFields: {
+        requestId?: string;
+        event?: string;
+        sourceIp: string | undefined;
+        objectId: string | undefined;
```

Comment:
How is a `sourceIp: string | undefined` different from `sourceIp?: string`?

Reply:
It's not. They're the same.